.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file needs an extension, probably `.json` since it looks like a JSON file. see [ESLint Docs](https://eslint.org/docs/user-guide/configuring#configuration-file-formats) regarding file names

Reply:
It is indeed, a json formatted file, but other .eslintrc files don't need an extension. I thik it may have to do with soft/hard spaces. I didn't take a look at it on my local machine yet.

Reply:
When not using an extension you're implying the file is deprecated.
Again from the docs above
>**Deprecated** - use .eslintrc, which can be either JSON or YAML.